schemas/SDMXStructureDataflow.xsd

```diff
@@ -25,19 +25,31 @@
 							<xs:documentation>Structure provides a reference to the data structure definition that defines the structure of all data for this flow.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
-					<xs:element name="DimensionConstraint" type="common:DimensionConstraintReferenceType" minOccurs="0">
+				</xs:sequence>
+				<xs:attribute name="urn" type="common:DataflowUrnType" use="optional"/>
+			</xs:restriction>
+		</xs:complexContent>
+	</xs:complexType>
+	
+	<xs:complexType name="DataflowType">
+		<xs:annotation>
+			<xs:documentation>DataflowType describes the structure of a data flow. A data flow is defined as the structure of data that will provided for different reference periods. If this type is not defined externally, then a reference to a data structure must be provided.</xs:documentation>
+		</xs:annotation>
+		<xs:complexContent>
+			<xs:extension base="DataflowBaseType">
+				<xs:sequence>
+					<xs:element name="DataConstraints" type="common:DataConstraintAttachmentType" minOccurs="0">
 						<xs:annotation>
-							<xs:documentation>DimensionConstraint provides a reference to the dimension constraint that defines the dimensions being used by this flow.</xs:documentation>
+							<xs:documentation>DataConstraints provides a list of references to the data constraints that define the allowed data content for this flow.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
-					<xs:element name="DataConstraints" type="common:DataConstraintAttachmentType" minOccurs="0">

```

Comment:
Cannot find the `common:DataConstraintAttachmentType` definition. Should be in SDMXCommonReferences.xsd

Reply:
It is in SDMXCommonReferences.xsd on line 1552

Reply:
OK, thanks! It was not in the PR, sorry!